### DISCUSSION TITLE
Will you be around in 5 or 10 years?

### DISCUSSION THREAD

**Discussion by olbackuper** [PythonNest/PyNest#76] [Q&A]
I'm excited to see a NestJS equivalent in Python. I like FastAPI but I think PyNest really adds to it with its modular design.

I am about to re-write a proof of concept, hacked Python codebase and am researching frameworks that would help me with this task. I found PyNest and now I'm wondering if this framework will exist in 5 years from now? And what about 10 years from now?
I realize that I can't expect such guarantees. I am simply asking for an honest opinion and how the project is backed (if at all), and if any companies are betting on it.

Also, I'm wondering how hard it would be to switch from PyNest to just FastAPI, in case PyNest won't be maintained anymore in 5-10 years...

**Comment by ItayTheDar** [comment]
Hi @olbackuper, PyNest was born out of a need of my company Lemonade to build large-scale, DI First, Micro-services in Python. As for Now, Lemonade, which is a 1B$ Company has 2 PyNest-based microservices deployed to production and used by more than 30 engineers at the company. I'm committed to maintaining PyNest and establishing a full ecosystem around it.

About you last question, The migration from PyNest to fastAPI should be easy, since PyNest is eventually just another layer on top of fast API (the same way as fastapi is a layer on top of Starlett). 

If you need any support or assistance in your poc, you are more than welcome to set up a meeting with me, or just send me an email - itay.dar@lemonade.com.

Good Luck!
